modules/tinymce/src/plugins/importcss/main/ts/core/ImportCss.ts

```diff
@@ -34,11 +34,11 @@ const removeCacheSuffix = function (url: string) {
   return url;
 };
 
-const isSkinContentCss = function (editor: Editor, href: string) {
-  const settings = editor.settings, skin = settings.skin !== false ? settings.skin || 'oxide' : false;
+const isSkinContentCss = (editor: Editor, href: string) => {
+  const skin = Settings.getSkin(editor);
 
   if (skin) {
-    const skinUrl = settings.skin_url ? editor.documentBaseURI.toAbsolute(settings.skin_url) : EditorManager.baseURL + '/skins/ui/' + skin;
+    const skinUrl = Settings.getSkinUrl(editor) ? editor.documentBaseURI.toAbsolute(Settings.getSkinUrl(editor)) : EditorManager.baseURL + '/skins/ui/' + skin;
```

Comment:
Hm. Maybe worth assigning `Settings.getSkinUrl(editor)` to a variable? It's only used twice, but that line is very long and I figure it might make it a bit more readable.